lib/matplotlib/cbook/__init__.py

```diff
@@ -13,6 +13,12 @@
 from six.moves import xrange, zip
 from itertools import repeat
 import collections
+
+try:
+    from collections.abc import MappingView
+except ImportError:
+    MappingView = None
```

Comment:
2.7 has mappingview (https://docs.python.org/2.7/library/collections.html#collections.MappingView) so I don't know when it can ever be triggered.

(My guess is that below, the test on whether we're running Py3 should just be removed -- we may as well sanitize the case where someone passes `dic.itervalues()` on Py2 too (well I always thought the whole idea was misguided but let's at least be consistent across Python versions.))